src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -33,6 +33,11 @@ public RpcWorkerConfigFactory(IConfiguration config, ILogger logger, ISystemRunt
             _environment = environment ?? throw new ArgumentNullException(nameof(environment));
             _metricsLogger = metricsLogger;
             WorkersDirPath = Path.Combine(Path.GetDirectoryName(new Uri(typeof(RpcWorkerConfigFactory).Assembly.CodeBase).LocalPath), RpcWorkerConstants.DefaultWorkersDirectoryName);
+            if (!Directory.Exists(WorkersDirPath))
+            {
+                // Site Extension. Default to parent directory
+                WorkersDirPath = Path.Combine(Directory.GetParent(Path.GetDirectoryName(new Uri(typeof(RpcWorkerConfigFactory).Assembly.CodeBase).LocalPath)).FullName, RpcWorkerConstants.DefaultWorkersDirectoryName);
```

Comment:
Same comment in the other one about reusing `Path.GetDirectoryName(new Uri(typeof(RpcWorkerConfigFactory).Assembly.CodeBase).LocalPath)`